typescript/package.json

```diff
@@ -1,22 +1,42 @@
 {
-    "//": "TODO TODO Add package name, description, license, etc. here",
+    "name": "kurtosis-engine-api-lib",
     "version": "0.1.0",
     "main": "./build/index",
+    "description": "This repo contains a Typescript client for communicating with the Kurtosis Engine server, which is responsible for creating and destroying Kurtosis Enclaves.",
     "types": "./build/index",
     "scripts": {
-        "build": "tsc",
+        "build": "tsc && cp -R src/kurtosis_engine_rpc_api_bindings build/",
         "test": "ts-mocha -p tsconfig.json 'test/**/*.ts'"
     },
     "files": [
         "build"
     ],
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/kurtosis-tech/kurtosis-engine-api-lib.git"
+    },
+    "keywords": [
+        "kurtosis",
+        "engine"
+    ],
+    "author": "Kurtosis Technologies Inc <accounts@kurtosistech.com> (https://www.kurtosistech.com/)",
```

Comment:
Oo `accounts@` is actually the email we use to sign up for services (e.g. Quip); you'll want `support@kurtosistech.com` instead